### PR TITLE
Updated circleci config to only trigger benchmark by label action:run-benchmark and to run performance nightly only once a week

### PR DESCRIPTION
**Pull request by filipecosta90**
- @rafie before merging we need to ensure `CIRCLE_CI_SECRET` is present on repo
- @gkorland as discussed moving nightly of performance to once a week

### CODE REVIEW

File: .github/workflows/trigger-benchmark.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  haslabel:
+    name: analyse labels
+    runs-on: ubuntu-latest
+    outputs:
+      benchmark: ${{ steps.haslabel.outputs.labeled-run-benchmark }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Labeled with run-benchmark
+        id: haslabel
+        uses: DanielTamkin/HasLabel@v1.0.4
```
Comment (chayim) [inline .github/workflows/trigger-benchmark.yml:17]: Thanks for this. Both VecSim + module-tests do this by hand. Cool to see an action for this.

File: .github/workflows/trigger-benchmark.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        with:
+          contains: 'run-benchmark'
+  perf-ci:
+    name: Trigger CI benchmarks
+    needs: haslabel
+    if: needs.haslabel.outputs.benchmark
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: curl-circle-ci
```
Comment (chayim) [inline .github/workflows/trigger-benchmark.yml:27]: https://github.com/marketplace/actions/trigger-circleci-pipeline

File: .github/workflows/trigger-benchmark.yml (hunk 1 above)
Comment (filipecosta90) [reply .github/workflows/trigger-benchmark.yml:17]: :+1:

File: .github/workflows/trigger-benchmark.yml (hunk 2 above)
Comment (filipecosta90) [reply .github/workflows/trigger-benchmark.yml:27]: @chayim with the following configuration ( as exemplified in ) this GH action is "forcing you" to use specific parameters and not the ones you want/need:

image

IMHO I would avoid using the action before it maturing to the point in which you can use it without issues/properly setup the parameter names we want instead of those. 
TLDR: reverting back to the old curl

Comment (chayim) [reply .github/workflows/trigger-benchmark.yml:27]: WFM

**Review by chayim — APPROVED** [review]
